src/main/java/org/kaazing/monitoring/client/impl/MetricsCollectorAgrona.java

```diff
@@ -40,9 +40,13 @@
     private static int COUNTER_LABELS_BUFFER_LENGTH = 32 * 1024 * 1024;
     private static int COUNTER_VALUES_BUFFER_LENGTH = 1024 * 1024;
 
-    private static final String MONITOR_DIR_NAME = "\\kaazing";
+    private static final String MONITOR_DIR_NAME = "/kaazing";
```

Comment:
This name is still operating system dependent.  We need to determine the directory in a platform independent way.